habitat_baselines/common/utils.py

```diff
@@ -174,3 +177,55 @@ def generate_video(
         tb_writer.add_video_from_np_images(
             f"episode{episode_id}", checkpoint_idx, images, fps=fps
         )
+
+
+def image_resize_shortest_edge(img: np.ndarray, size: int):
+    """Resizes an img so that the shortest side is length of size.
+
+    Args:
+        img: the array object that needs to be resized
+        size: the size that you want the shortest edge to be resize to
+    Returns:
+        the resized array
+    """
+    h, w = img.shape[:2]
+    if w > h:
+        percent = size / h
+    else:
+        percent = size / w
+    h *= percent
+    w *= percent
+    h = int(h)
+    w = int(w)
+    img = cv2.resize(img, (h, w), interpolation=cv2.INTER_AREA)
```

Comment:
Why not do this on the GPU with https://pytorch.org/docs/stable/nn.functional.html?highlight=interpolate#torch.nn.functional.interpolate ?